docs/devcontainer-spec-support.md

```diff
@@ -0,0 +1,72 @@
+| 🔴                            | `workspaceMount`                      | Overrides the default local mount point for the workspace when the container is created.                               | -                                                      |
+| 🔴                            | `workspaceFolder`                     | Default path to open when connecting to the container.                                                                 | -                                                      |
+| **Docker Compose**            |                                       |                                                                                                                        |
+| 🔴                            | `dockerComposeFile`                   | Path to a Docker Compose file related to the `devcontainer.json`.                                                      | [#236](https://github.com/coder/envbuilder/issues/236) |
+| 🔴                            | `service`                             | Name of the Docker Compose service to which supporting tools should connect.                                           | [#236](https://github.com/coder/envbuilder/issues/236) |
+| 🔴                            | `runServices`                         | Docker Compose services to automatically start.                                                                        | [#236](https://github.com/coder/envbuilder/issues/236) |
+| **Lifecycle scripts**         |                                       |                                                                                                                        |
+| 🔴                            | `initializeCommand`                   | Command to run on the host machine when creating the container.                                                        | [#395](https://github.com/coder/envbuilder/issues/395) |
+| 🟢                            | `onCreateCommand`                     | Command to run inside container after first start.                                                                     |                                                        |
+| 🟢                            | `updateContentCommand`                | Command to run after `onCreateCommand` inside container.                                                               |                                                        |
+| 🟢                            | `postCreateCommand`                   | Command to run after `updateContentCommand` inside container.                                                          |                                                        |
+| 🟢                            | `postStartCommand`                    | Command to run each time the container is started.                                                                     |                                                        |
```

Comment:
We may want to link to an issue showing it is not logged in Coder? Feels a bit strange since this is an individual project

Reply:
I'll add an asterisk (*) for now.